daemon/api_notices.go

```diff
@@ -130,7 +131,12 @@ func getNotices(c *Command, r *http.Request, user *auth.UserState) Response {
 	if notices == nil {
 		notices = []*state.Notice{} // avoid null result
 	}
-	return SyncResponse(notices)
+
+	noticeInfos := make([]*noticeInfo, 0, len(notices))
```

Comment:
I believe this conversion will mean the "avoid null result" above is no longer needed, as this will always `make` a non-null result.

Reply:
Good point, Thank you! updated